pull/README.md

```diff
@@ -0,0 +1,282 @@
+# Remote Provider Pull
+
+The `pull` addon was recently a core command but has been pulled out due as it is sometimes 
+quicker to get updates to just the addon as opposed to the whole platform.
+
+`fin pull` command allows you to easily bring your assets: db, files, and code from your hosting environment
+down to your local environment without manually running through the steps.
+
+## Installation
+
+```bash
+fin addon install --global pull
+```
+
+## Requirements
+
+### Acquia
+
+To interact with Acquia through Drush, the `SECRET_ACQUIA_CLI_SECRET` and
+`SECRET_ACQUIA_CLI_KEY` are required. Additionally, access to the Acquia Cloud API
+is required. If access is not allowed, fallback to using the [Drush](https://docs.docksal.io/fin/fin-pull/#drupal-drush) method.
```

Comment:
"To interact with Acquia through the Acquia CLI, ..."

Drush to Acquia only requires an SSH key. It's the Acquia CLI that requires the additional secret and key.